examples/03_connectivity/plot_multi_subject_connectome.py

```diff
@@ -89,26 +89,33 @@ def plot_matrices(cov, prec, title):
 gsc = GroupSparseCovarianceCV(verbose=2)
 gsc.fit(subject_time_series)
 
-from sklearn import covariance
-gl = covariance.GraphLassoCV(verbose=2)
-gl.fit(np.concatenate(subject_time_series))
-
+from nilearn.connectome import ConnectivityMeasure
+from sklearn.covariance import GraphLassoCV
+gl_covariance_measure = ConnectivityMeasure(GraphLassoCV(verbose=2),
+                                            kind='covariance')
+gl_covariance = gl_covariance_measure.fit_transform(
+    [np.concatenate(subject_time_series)])[0]
+
+gl_precision_measure = ConnectivityMeasure(GraphLassoCV(verbose=2),
+                                           kind='precision')
+gl_precision = gl_covariance_measure.fit_transform(
+    [np.concatenate(subject_time_series)])[0]
```

Comment:
This is also adding a lot of lines. I am a bit worried that it actually makes all this harder to explain / learn.